apps/accounts/views.py

```diff
@@ -538,6 +539,15 @@ def reset_password(
 
         if user:
             validlink = True
+
+            # Add mobile deep link URL if on mobile and format provided
+            if (request.user_agent.is_mobile or request.user_agent.is_tablet) and mobile_reset_url_format:
+                data['mobile_reset_url'] = mobile_reset_url_format.format(
+                    uid_base36=uidb36,
+                    token=token
+                )
+                data['appstore_url'] = htk_setting('HTK_APPSTORE_URL')
```

Comment:
Please add:

`htk/apps/mobile/constants/defaults.py`

The constant should have a default value of `None`

`HTK_APPSTORE_URL = None`

(New app under this dir: https://github.com/hacktoolkit/django-htk/tree/master/apps)

Then, include the import here:
 https://github.com/hacktoolkit/django-htk/blob/d231b9da22f2570ac81b08c49d359b56122523a2/constants/defaults.py#L104-L105